packages/chains/src/polygon/assets.ts

```diff
@@ -395,6 +395,20 @@ export const assets: SupportedAsset[] = [
     decimals: 18,
     oracle: OracleTypes.ChainlinkPriceOracleV2,
   },
+  {
+    symbol: assetSymbols.JCOP,
```

Comment:
Actually, we don't support these, because we don't have oracles. Let's remove them please, otherwise oracle-monitor will puke -- and we'll show these in the UI